metrics/run_all_metrics.sh

```diff
@@ -16,6 +16,20 @@
 
 set -e
 
+# Check arguments
+case ${RUNTIME#[-+]} in
+    ''|*[!0-9]* )
+        echo Variable RUNTIME must be set to a valid integer
```

Comment:
For consistency, I'd recommend sourcing `lib/common.bash` then just calling `die "Variable RUNTIME must be set to a valid integer`".